Xamarin.Essentials/mdoc.targets

```diff
@@ -132,6 +133,32 @@
       <Code Type="Fragment" Language="cs"><![CDATA[
         foreach (var file in Directory.EnumerateFiles(DocsRoot, "*.xml", SearchOption.AllDirectories)) {
           var xdoc = XDocument.Load(file);
+
+          // special case for Android resources: don't process
```

Comment:
After including the support libraries and resources, we started getting massive files with all the possible resources in the world - we don't really care (I think)